tests/DocBlock/AnnotationTest.php

```diff
@@ -309,6 +309,30 @@ public function provideTypeParsingCases()
                 '/** @var array<string|int, string>',
                 ['array<string|int, string>'],
             ],
+            [
+                '/** @var Foo::*',
+                ['Foo::*'],
+            ],
+            [
+                '/** @var Foo::A',
+                ['Foo::A'],
+            ],
+            [
+                '/** @var Foo::A|Foo::B',
+                ['Foo::A', 'Foo::B'],
+            ],
+            [
+                '/** @var Foo::A*',
+                ['Foo::A*'],
+            ],
+            [
+                '/** @var array<Foo::A*>|null',
+                ['array<Foo::A*>', 'null'],
+            ],
+            [
+                "/** @var null|true|false|1|1.5|'a'|'b'",
```

Comment:
@Nyholm I'm not familiar with Psalm: does it support all these types? Does it support double quoted strings?

Reply:
Great question. I had to look it up. 

Yes, it does: https://psalm.dev/r/6564c220dc

Reply:
Yes, it all looks good. 

https://psalm.dev/r/66146836e6

Reply:
Cool, I updated the test case with a double quoted string, thanks for checking!